rplugin/python3/deoplete/util.py

```diff
@@ -52,6 +55,43 @@ def globruntime(runtimepath, path):
     return ret
 
 
+def find_rplugins(context, source):
+    """Search for base.py or *.py
+
+    Searches $VIMRUNTIME/*/rplugin/python3/deoplete/$source[s]/
+    """
+    rtp = context.get('runtimepath', '').split(',')
+    if rtp:
```

Comment:
👍  This is a habit I have because the function is a generator, but it's only a problem if it returns a value.  I'll change it.